ppb/scenes.py

```diff
@@ -48,7 +48,9 @@ def add(self, game_object: Hashable, tags: Iterable[Hashable]=()) -> None:
         if isinstance(tags, (str, bytes)):
             raise TypeError("You passed a string instead of an iterable, this probably isn't what you intended.\n\nTry making it a tuple.")
         self.all.add(game_object)
-        self.kinds[type(game_object)].add(game_object)
+
+        for kind in type(game_object).mro():
```

Comment:
This is a tricky thing: mro here is going to include `BaseSprite`, `Sprite` (the ABC), and `object`. Are you intending for that to happen or would you rather explicitly exclude those?

Reply:
I couldn't think of any reasons to exclude them (except maybe `object`?), and there are no documented exceptions about what are valid `kinds`.

Reply:
The question is: what's the difference between `self.all` and `self.kinds[object]` under this new functionality?